notes/src/components/NoteView.js

```diff
@@ -0,0 +1,105 @@
+import React, { Component } from 'react';
+import axios from '../../node_modules/axios';
+import { getNotes } from '../actions';
+
+class NoteView extends Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            note: {},
+            noteTitleInput: '',
+            noteBodyInput: '',
+        }
+    }
+    
+
+    componentDidMount() {
```

Comment:
Very nice refactor! Once set up, Redux makes things a bit easier to put together, would you agree?